wolfssl/openssl/sha.h

```diff
@@ -107,8 +111,20 @@ typedef WOLFSSL_SHA_CTX SHA_CTX;
  * to Sha224, is expected to also be 16 byte aligned addresses.  */
 typedef struct WOLFSSL_SHA224_CTX {
     /* big enough to hold wolfcrypt Sha224, but check on init */
+#if defined(WOLFSSL_IMXRT1170_CAAM)
```

Comment:
Perhaps you can hijack `CTX_SHA2_HW_ADDER` instead to avoid duplication of this compatibility sizing? Comment applies to several places.